papis/commands/update.py

```diff
@@ -183,34 +243,49 @@ def run_set(
     for orig_key, orig_value in to_set:
         key, vformat = process_format_pattern_pair(orig_key, orig_value)
         value = format(vformat, document, default=str(vformat))
-        value = try_parsing_str(key, value)
-
-        if isinstance(value, int) and key_types.get(key) is str:
-            value = str(value)
-        if key == "notes" and isinstance(value, str):
-            # TODO: handle renames/deletions of files on disk
-            document[key] = normalize_path(value)
-            logger.warning(
-                "Document note renamed in the info.yaml file. This does not "
-                "rename any files on disk."
-            )
-        elif key == "files" and isinstance(value, list):
-            # TODO: handle renames/deletions of files on disk
-            document[key] = []
-            for file in value:
-                if isinstance(file, str):
-                    document[key].append(normalize_path(file))
-                else:
-                    document[key].append(value)
```

Comment:
@jghauser I don't quite recall how this was supposed to work? It seems to just overwrite the whole files list + not sure what that `isinstance(file, str)` is supposed to do..